server/events.go

```diff
@@ -138,11 +138,14 @@ const DisconnectEventMsgType = "io.nats.server.advisory.v1.client_disconnect"
 // updates in the absence of any changes.
 type AccountNumConns struct {
 	TypedEvent
-	Server     ServerInfo `json:"server"`
-	Account    string     `json:"acc"`
-	Conns      int        `json:"conns"`
-	LeafNodes  int        `json:"leafnodes"`
-	TotalConns int        `json:"total_conns"`
+	Server        ServerInfo `json:"server"`
+	Account       string     `json:"acc"`
+	Conns         int        `json:"conns"`
+	LeafNodes     int        `json:"leafnodes"`
+	TotalConns    int        `json:"total_conns"`
+	Sent          DataStats  `json:"sent"`
```

Comment:
When is this sent out? I can't recall off top of my head.. Is it only when a user requests?
We need to optimize the JSON at some point, so just curious.

Reply:
this is sent when a connection is added or removed as well as every 30 seconds.
The message itself is account specific and sent on these subjects
```
	accConnsEventSubjNew     = "$SYS.ACCOUNT.%s.SERVER.CONNS"
	accConnsEventSubjOld     = "$SYS.SERVER.ACCOUNT.%s.CONNS" // kept for backward compatibility
```

Reply:
@matthiashanel can this be accesed from within the account? We have the `SYS.REQ.SERVER.PING.CONNZ` in the account, but this would be nicer to have

Reply:
Not at the moment but its on his list to add AFAIK.

Reply:
The conns envelope format does not appear to be identical to the other requests. 
So I'd re expose it with the same envelope, used everywhere ,under $SYS.REQ.ACCOUNT.%s.STATZ as well as $SYS.REQ.ACCOUNT.PING.STATZ inside the system account. 
I would also import $SYS.REQ.ACCOUNT.%s.STATZ as $SYS.REQ.ACCOUNT.PING.STATZ into the actual account. 

Provided we already import "$SYS.REQ.SERVER.PING.CONNZ" as "$SYS.REQ.ACCOUNT.PING.CONNZ" into the account, it might be easier to differentiate as well. 

wdyt @ripienaar

Reply:
Would be cool fusing as we have a statsz already with other content ? Something something  connz would be best but yeah it’s not a good situation :(